src/model.js

```diff
@@ -2644,8 +2644,8 @@ class Model {
           options.returning = true;
         }
       }
-
-      if (options.ignoreDuplicates && ['mssql', 'db2', 'oracle'].includes(dialect)) {
+      if (options.ignoreDuplicates && this.sequelize.dialect.supports.inserts.ignoreDuplicates === '' &&
```

Comment:
Holy mother of Jesus, I didn't expect that thing to be a string. Since empty strings are false, I wonder if we should go with

```
options.ignoreDuplicates && !this.sequelize.dialect.supports.inserts.ignoreDuplicates
```

Why do we consider onConflictDoNothing suddenly?

Reply:
Without adding onConflictDoNothing, I see this test case for postgresql was failing.
```
test/integration/model/bulk-create.test.js
 1) should support the ignoreDuplicates option
```

The test case enables ignoreDuplicates option under a check if either of property;  ignoreDuplicates or onConflictDoNothing is non-null.  Hence validating against only supports.inserts.ignoreDuplicates throws an error. 

`Error: postgres does not support the ignoreDuplicates option.`

I modified to use ! for string empty check.
on the CI using latest version, can we add the lower version 18.4 in a separate PR?